source/includes/fundamentals/code-examples/builders.cs

```diff
@@ -12,25 +12,41 @@ public static void Main(string[] args)
     {
         Setup();
 
-        // Sample builders definitions
+        // Creates a filter for all documents with a "season" value of "Spring"
         var filter = Builders<Flower>.Filter.AnyEq(flower => flower.Season, "spring");
+
+        // Creates a projection to include the "name" and "price" fields, while
+        // excluding the "id" field
         var projection = Builders<Flower>.Projection.Include("Name").Include("Price").Exclude("Id");
+
+        // Creates a sort object that sorts results by ascending price, and, in
+        // the case the multiple documents have the same price, sorts in reverse
+        // alphabetical order by category
```

Comment:
S: just to shorten a little

```suggestion

        // Creates instructions to sort results by ascending price and to sort results with the same price in descending category order
```